<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdiPlanning</class>
 <widget class="QWidget" name="MdiPlanning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Planning des réservations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidgetPlanning">
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <row>
          <property name="text">
           <string>Disponible</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Occupée</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Reservée</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV4</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Lundi 8</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mardi 9</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mercredi 10</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Jeudi 11</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Vendredi 12</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Samedi 13</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Dimanche 14</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Période allant du :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditDateDebut">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>à</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEditDateFin">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="btn_setPeriodeReserv">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QFormLayout" name="formLayout_6">
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Durée :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditDuree">
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Planning des occupations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidgetPlanningOccup">
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <row>
          <property name="text">
           <string>Disponible</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Occupée</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Reservée</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>CV4</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Lundi 8</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mardi 9</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mercredi 10</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Jeudi 11</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Vendredi 12</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Samedi 13</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Dimanche 14</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_12">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Période allant du :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateDebutOccup">
                  <property name="minimumSize">
                   <size>
                    <width>160</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>à</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QFormLayout" name="formLayout_18">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEditDateFinOccup">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="btn_setPeriodeOccup">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QFormLayout" name="formLayout_19">
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Durée :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditDureeOccup">
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFacturesDeCeJour">
      <attribute name="title">
       <string>Factures de ce jour</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidgetFacturesDeJour">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>130</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>130</number>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Facture</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Client</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Chambre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date d'arrivée</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date de départ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Durée</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Montant total</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Montant payé</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Montant restant</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Matricule agent</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_14">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Montant encaissé total :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditMontantEncaisse">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Montant restant total :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditMontantRestantDeCeJour">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_imprimerFactureDeCeJour">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Imprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFacturesNonReglees">
      <attribute name="title">
       <string>Factures non réglées</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidgetFactures">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>120</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>120</number>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Facture</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Client</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Chambre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date d'arrivée</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date de départ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Durée</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Montant total</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Montant payé</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Montant restant</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Matricule agent</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_9">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Montant restant total :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditMontantRestant">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QFormLayout" name="formLayout_10">
         <property name="formAlignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_reglerFacture">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>23</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Régler la facture</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_imprimerFacture">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>23</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Imprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabActivites">
      <attribute name="title">
       <string>Activités</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_21">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_8">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditMontantTotalEncaisse">
             <property name="maximumSize">
              <size>
               <width>133</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Montant total encaissé :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QFormLayout" name="formLayout_20">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Montant total retiré :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditMontantTotalRetire">
             <property name="maximumSize">
              <size>
               <width>133</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QFormLayout" name="formLayout_7">
         <property name="formAlignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_imprimerRecu">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>23</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Imprimer</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_cloturer">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>23</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Clôturer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="tableViewActivites">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>210</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>210</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
